packages/console/src/utilities/validator.ts

```diff
@@ -14,3 +14,17 @@ export const uriValidator = (verifyBlank = true) => {
     return true;
   };
 };
+
+export const uriOriginValidator = ({ allowBlank = true }) => {
```

Comment:
origin validator should not allow blank

Reply:
this is a confusing naming, should rename back to verifyBlank, updated